test/helpers/tenant/secrets.go

```diff
@@ -4,10 +4,12 @@ package tenant
 
 import (
 	"context"
+	b64 "encoding/base64"
```

Comment:
```suggestion
	"encoding/base64"
```

Reply:
Should be probably the same. Would spoil this PR a bit, though.

Reply:
fixed